schema.graphql

```diff
@@ -57,7 +57,7 @@ type Query {
 input CreateUserInput {
   id: String!
   email: String!
-  displayName: String
+  displayName: String!
```

Comment:
This shouldn't change. I'll comment in the Rust code where I disagree on the change.